chrome/content/otr.js

```diff
@@ -276,6 +284,18 @@ var otr = {
     return human.readString();
   },
 
+  base64encode: function(data, dataLen) {
+    let buf = ctypes.char.array(Math.floor(((dataLen+2)/3)*4 + 1))();
+    let size = libOTR.otrl_base64_encode(buf, data, dataLen);
+    return buf.readString();
```

Comment:
Yes. I added one more character to `buf`. It should be a null terminated string now. Thanks!